tests/class.test.tsx

```diff
@@ -141,94 +141,3 @@ it('inherited class without methods', async () => {
   fireEvent.click(getByText('button'))
   await findByText('count: 1')
 })
-
-it('class with a method', async () => {
-  class CountClass {
-    public count: number
-    constructor() {
-      this.count = 0
-    }
-    public doubled() {
-      return this.count * 2
-    }
-  }
-
-  const obj = proxy(new CountClass())
-
-  const Counter: React.FC = () => {
-    const snapshot = useProxy(obj)
-    return (
-      <>
-        <div>count: {snapshot.count}</div>
-        <div>doubled: {snapshot.doubled()}</div>
```

Comment:
This looks nice, but as snapshot.doubled wouldn't change, it will not trigger re-renders.
The only reason this example works is `snapshot.count` is used in the previous line.